### DISCUSSION TITLE
T2T-CHM13 reference genome

### DISCUSSION THREAD

**Discussion by gavinmonahan** [broadinstitute/seqr#2796] [Q&A] [answered]
Hi,

I just wanted to check if there will be support for the T2T-CHM13 reference genome for the CMG seqr or local seqr? We are just planning on what to do with our unsolved cases and would like to see if the new reference genome would help.

Thanks,
Gavin

**Comment by hanars** [comment, marked as answer]
Hi Gavin,

That is a very reasonable request, but we have no near-term plans to support this reference genome. CMG data sequenced at the Broad is currently all delivered on build 38 so there is no real impetus to do this work. At some point we assume the Broad GP will switch which reference genome they deliver and seqr will update to support it at that time, but we have no real sense of when that would be. If we get enough interest from outside the Broad we might consider implementing it earlier, but that would still be mid-2023 at the soonest

**Reply by gavinmonahan** [reply]
Makes perfect sense, thank you!

**Reply by melnel000** [reply]
For those of us working in Africa, moving to T2T-CHM13 is something we are keen to explore soon. My research group would really welcome the ability to analyze T2T-CHM13 data in seqr.

**Comment by grabear** [comment]
Hi seqr team.  Just curious if grch38 is still the standard here and if T2T support is on the roadmap?

**Reply by hanars** [reply]
Hey thanks for checking in! Unfortunately, our answer from back in 2022 still stands, as we have not started receiving any T2T data at this time and have not had any real pressing motivation to start this support work. I would also note that seqr relies heavily on filtering data based on population frequencies, so it is unlikely that we would start work on T2T support until gnomad or a comparable population database was well supported for this build.
